components/modal.css

```diff
@@ -0,0 +1,19 @@
+.modal-wrapper {
+  z-index: 1;
+  background-color: rgba(0, 0, 0, .5);
```

Comment:
Suggestion: You can replace rgba with color variable to increase reusability and redundancy of the color.

Reply:
This is one-off usage of this color, hence directly used rgba value.